pom.xml

```diff
@@ -318,6 +331,7 @@
                         <format>html</format>
                         <format>xml</format>
                     </formats>
+                    <check/>
```

Comment:
what is this?

Reply:
Check tag is required by the plugin config, in this case it doesn't do anything. This tag makes IDEA not consider it an error anymore